LoggingPreferences/LoggingPreferences.json

```diff
@@ -8,9 +8,9 @@
     },
     "definitions": {
         "keystrokeMaskEnabled_i": {
-            "summary": "Whether keystroke mask is enabled. If `1` (true), then any system that logs keystrokes must mask the actual keystroke being used. Default value is `0` (false).",
-            "type":"integer",
-            "example": 0
+            "summary": "Whether keystroke mask is enabled. If `true`, then any system that logs keystrokes must mask the actual keystroke being used. Default value is `false`",
+            "type":"boolean",
```

Comment:
Please remove "keystrokeMaskEnabled_i" and use "keystrokeMaskEnabled_b" for "isKeystrokeMaskEnabled".

Reply:
Updated Changes as per your review comments

Reply:
looks good